packages/app-project/stores/User/User.js

```diff
@@ -33,10 +33,18 @@ const User = types
 
   .actions(self => ({
     clear() {
-      self.id = null
-      self.display_name = null
-      self.login = null
-      self.loadingState = asyncStates.success
+      const loggedOutUser = {
+        id: null,
+        display_name: null,
+        login: null,
+        loadingState: asyncStates.success,
+        personalization: {
+          projectPreferences: {
+            loadingState: asyncStates.success
```

Comment:
Without this, the classifier never loads for anonymous volunteers. 🤔

Reply:
I think it could be changed by fetching preferences in a hook, similar to how the user is fetched from Panoptes.